notebook/static/notebook/js/actions.js

```diff
@@ -437,7 +437,18 @@ define(function(require){
                 $('.header-bar').toggle();
                 events.trigger('resize-header.Page');
             }
-        }
+        },
+        'escape': {
```

Comment:
OK, let's get some broader feedback from other folks.

On Mon, Oct 26, 2015 at 1:39 PM, Matthias Bussonnier <
notifications@github.com> wrote:

> In notebook/static/notebook/js/actions.js
> <https://github.com/jupyter/notebook/pull/657#discussion_r43026425>:
>
> > @@ -437,7 +437,18 @@ define(function(require){
> >                  $('.header-bar').toggle();
> >                  events.trigger('resize-header.Page');
> >              }
> > -        }
> > +        },
> > +        'escape': {
>
> I split my vote 0.3 for Brian, 0.7 for Jon. Esc is naturally what people
> hit a lot of time when they don't know where they are I would expect Esc*(many
> times) to bring me to one particular state regardless of where I am.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/notebook/pull/657/files#r43026425>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com